app/assets/scss/application.scss

```diff
@@ -103,11 +103,11 @@ $path: "/suppliers/static/images/";
 // app-specific.
 ins {
   text-decoration: none;
-  background-color: #cce0d6;
+  background-color: #acf2bd;
```

Comment:
We shouldn't use custom colour codes here, probably. Can we using existing colours imported by one of our toolkits?

If we can't, we should at least define these as sass variables so that we can give them a semantic name.

Reply:
We can't use any existing colours because they do not follow any existing pattern as far as I am aware. I have created a variables sass file and also created a trello ticket to get this moved: https://trello.com/c/a90iPGOH/117-move-ins-del-to-frontend-toolkit.